taas_api/client.py

```diff
@@ -60,9 +60,20 @@ def get_balances(self):
     def get_orders(self):
         return self.get(path=f"/api/orders/")
 
-    def place_order(self, request: PlaceOrderRequest):
-        if not isinstance(request, PlaceOrderRequest):
-            raise ValueError(f"Expecting request to be of type {PlaceOrderRequest}")
+    def place_multi_order(self, request: data.PlaceMultiOrderRequest):
```

Comment:
out of the loop here, but are we replacing place_order entirely with place_multi_order rather than adding a functionality to place a multi-order on top of the current functionality of placing a normal order?